docs/maintainer/infrastructure.md

```diff
@@ -469,6 +469,159 @@ uploaded from the new feedstock, by design.
+
+On pushes to `main`:
+
+- Create and initialize the new feedstocks.
+
+Authenticated services involved:
+
+- Github, with permissions for:
+  - Repository creation
+- Azure Pipelines
+- Travis CI
+- Other CI providers?
+
+### Feedstock changes
+
+A feedstock can receive changes for several reasons.
+
+Pushes to `main` or other branches:
+
+- The automated initialization commits following approval in `staged-recipes`.
+  These are generated by `conda-smithy`.
+- Automated maintenance commits triggered from `admin-migrations`.
+- Rerender requests are handled by instances of `conda-forge/webservices-dispatch-action` and triggered by the admin web service.
+
+Automatic pull requests can be opened by...
+
+- `@conda-forge-linter`, responding to some issues with titles like `@conda-forge-admin, please...`.
+- `@regro-cf-autotick-bot`, handling migrations and new versions being available.
+
+...and closed by:
+
+- `conda-forge/automerge-action`, if labeled accordingly.
+
+On an open pull request:
+
+- The building pipelines (more [below](#package-building)).
+- The linter is provided by `conda-smithy recipe-lint`, run by `@conda-forge-linter`.
+- The `@conda-forge-admin, please...` command comments, answered by `@conda-forge-linter`.
+
+On issues:
+
+- `@conda-forge-admin, please...` command issues, handled by `@conda-forge-linter`.
+
+### Package building {#package-building}
+
+The pipelines that build conda packages are used for both pull requests and push events in `main` and other branches.
+The only difference is that the packages built during a pull request are not uploaded to the staging channel.
+Maintaining these up-to-date across all feedstocks involves several repositories:
+
+- `conda-smithy` is in charge of generating the CI pipelines themselves, together with the supporting scripts and configuration files.
+  These pipelines and scripts can rely on code and data defined in the repositories below.
+- `conda-forge-ci-setup-feedstock` provides the code needed to prepare and homogenize the CI runners across providers.
+  It also does some checks before the artifacts are uploaded to `cf-staging`.
+- `conda-forge-pinning-feedstock` defines which versions are supported for a number of runtimes and libraries, as well as the compilers used for certain languages and platforms.
+- `docker-images` builds the standardized container images for Linux runners.
+  This repository has additional authentication needs for Docker Hub, Quay.io.
+
+The pipelines can run on several CI providers supported by `conda-smithy`, including:
+
+- Azure DevOps Pipelines
+- Travis CI
+- Circle CI
+- Appveyor
+
+Registration of hooks and triggers is also done by the `conda-smithy` app.
+
+:::tip
+`conda-smithy` supports more CI providers.
+Check [its repository](https://github.com/conda-forge/conda-smithy) for more details.
+:::
+
+Authenticated services involved:
+
+- Anaconda.org uploads to `cf-staging`
+
+### Package validation and publication
```

Comment:
This might not be up-to-date. I think @beckermr adjusted this part of the infra not long ago.

Reply:
@beckermr, could you double-check whether this section needs adaptation?

Reply:
done!